### PR TITLE
Dedicated pipeline for cross-language tests

### PR DESCRIPTION
**Pull request by petrovicboban**

### CODE REVIEW

**Comment by edwardwliu** [comment]
This LGTM. Could we rename the `cross_language_test/` directory to just `tests/`?

File: cross_language_test/run.sh
Comment (edwardwliu) [inline cross_language_test/run.sh:1]: Could we update this path to tests/run.sh?

**Review by edwardwliu — APPROVED** [review]

**Comment by petrovicboban** [comment]
> This LGTM. Could we rename the `cross_language_test/` directory to just `tests/`?

Done.

File: cross_language_test/run.sh
Comment (petrovicboban) [reply cross_language_test/run.sh:1]: Done.

**Pull request merged**